packages/config/src/handlers/forms/validation.ts

```diff
@@ -0,0 +1,424 @@
+                  ({ name }) =>
+                    !(REQUIRED_FIELDS_IN_SECTION[sec.id] ?? []).includes(name)
+                )
+                .map(({ name }) => name)
+            )
+            return (
+              nonCustomfieldsInSection.filter(
+                (nonCustomField) =>
+                  !(OPTIONAL_FIELDS_IN_SECTION[sec.id] ?? []).includes(
+                    nonCustomField
+                  )
+              ).length === 0
+            )
+          },
+          (sec) => {
+            const nonCustomfieldsInSection = sec.groups.flatMap((group) =>
+              group.fields
+                .filter(({ custom }) => !Boolean(custom))
+                .filter(({ type }) => type !== 'DIVIDER')
+                .filter(
+                  ({ name }) =>
+                    !(REQUIRED_FIELDS_IN_SECTION[sec.id] ?? []).includes(name)
+                )
+                .map(({ name }) => name)
+            )
+            const unrecognizedFields = nonCustomfieldsInSection.filter(
+              (nonCustomField) =>
+                !(OPTIONAL_FIELDS_IN_SECTION[sec.id] ?? []).includes(
+                  nonCustomField
+                )
+            )
+            return {
+              message: `Use "custom" property to make a field custom. Unrecognized non custom fields "${unrecognizedFields.join(
+                ', '
+              )}" found`
+            }
+          }
+        )
+    )
+    .refine(
+      (sections) =>
+        sections.filter(({ id }) =>
+          REQUIRED_SECTIONS.includes(id as typeof REQUIRED_SECTIONS[number])
+        ).length >= 2,
+      {
+        message: `${REQUIRED_SECTIONS.map((sec) => `"${sec}"`).join(
+          ' & '
+        )} sections are required`
+      }
+    )
+    .refine(
+      (sections) => {
+        findDuplicates(
+          sections
+            .flatMap((sec) => [
+              ...(sec.mapping?.template ?? []).map(
+                ({ fieldName }) => fieldName
+              ),
+              ...sec.groups
+                .flatMap((group) => group.fields)
+                .map(({ mapping }) => mapping?.template?.fieldName)
+            ])
+            .filter((maybeName): maybeName is string => Boolean(maybeName))
+        ).length === 0
+      },
+      (sections) => {
+        const duplicateCertificateHandlebars = findDuplicates(
+          sections
+            .flatMap((sec) => [
+              ...(sec.mapping?.template ?? []).map(
+                ({ fieldName }) => fieldName
+              ),
+              ...sec.groups
+                .flatMap((group) => group.fields)
+                .map(({ mapping }) => mapping?.template?.fieldName)
+            ])
+            .filter((maybeName): maybeName is string => Boolean(maybeName))
+        )
+        return {
+          message: `All the certificate handlebars should be unique for an event. Duplicate handlebars for "${duplicateCertificateHandlebars.join(
+            ', '
+          )}"`
+        }
```

Comment:
@Zangetsu101 I fixed all the certificate handlebars in [this PR](https://github.com/opencrvs/opencrvs-core/pull/5785). But this validation still throws an error even if the duplicateCertificateHandlebars array is empty